packages/patternfly-4/react-core/src/components/List/List.tsx

```diff
@@ -1,27 +1,53 @@
 import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/List/list';
 import { css, getModifier } from '@patternfly/react-styles';
+import { Omit } from '../../helpers/typeUtils'
+
+export enum OrderType {
+  number = '1',
+  lowercaseLetter = 'a',
+  uppercaseLetter = 'A',
+  lowercaseRomanNumber = 'i',
+  uppercaseRomanNumber = 'I',
+}
 
 export enum ListVariant {
-  inline = 'inline'
+  inline = 'inline',
 }
 
-export interface ListProps extends React.HTMLProps<HTMLUListElement> {
+export enum ListComponent {
+  ordered = 'ol',
```

Comment:
Looks good.  My preference would be to name them by the component name.
`export enum ListComponent {
    ordered = 'ol',
    unordered = 'ul',
}`